pkg/gardenlet/controller/seed/seed_lease_control.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
```

Comment:
Can we have a unit test for this file?

Reply:
Just an update: I am trying, but it is starting to look really bad 😁